db/models/assessment.py

```diff
@@ -59,3 +64,18 @@ def register_application(application_id: str):
                 message="An assessment for this application already exists"
             )
         return assessment
+
+    @staticmethod
+    def update_status(assessment_id: str, status: str):
+        try:
+            assessment_id = AssessmentMethods.get_by_id(assessment_id)
+            assessment_id.compliance_status = status
+            db.session.commit()
+            assessment = Assessment(
```

Comment:
Could we just return this directly rather than assigning it first?

Reply:
Done